pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -378,8 +382,15 @@ func (nim *nodeInfoManager) tryUpdateCSINode(
 	driverNodeID string,
 	maxAttachLimit int64,
 	topology map[string]string) error {
-
-	nodeInfo, err := csiKubeClient.StorageV1beta1().CSINodes().Get(string(nim.nodeName), metav1.GetOptions{})
+	var nodeInfo *storagev1beta1.CSINode
+	var err error
+	if nim.nodeLister != nil {
+		nodeInfo, err = nim.nodeLister.Get(string(nim.nodeName))
+		klog.Infof("informer returned %v err: %v", nodeInfo, err)
+	} else {
+		// for standalone kubelet mode, return early
+		return nil
```

Comment:
I think we should do this even earlier (and log a message) when this pkg is initialized